importers/wikidata.py

```diff
@@ -37,19 +38,27 @@ def load_person_from_wikidata(wikidata_url):
     else:
         return {}
 
-
 @cache.dict()
 def load_person_from_wikipedia(wikidata_url, language):
     """Given a wikidata url, get information from wikipedia
     TODO: Allow a wikipedia URL as argument too
     TODO: Check language against valid list
     TODO: Image from wikipedia is different to that of wikidata"""
-    entity = get_entity_for_wikidata(wikidata_url)
+    
+    if 'wikipedia' not in wikidata_url:
```

Comment:
we should have a separate method for the two cases in this if statement - especially because we're planning on merging this repo into the main data importer repo shortly, and this file (wikidata.py) is shared between the two.

This method is named really badly - what it does is take a wikidata URL, and go to the related wikipedia url in a specific language and get a description. If we want another method to get the description from a given _wikipedia_ page, we should have another method that does specifically that.

Reply:
I have updated the methods cleaning a bit